src/toil_scripts/batch_alignment/README.md

```diff
@@ -1,47 +1,146 @@
 
-    1. Toil     pip install toil
-    2. S3AM     pip install --pre s3am (optional, for upload of BAMFILE to S3)
+    1. Toil         pip install toil
+    2. S3AM         pip install --pre s3am (optional, needed for uploading output to S3)
+
+## Inputs
+
+The BWA pipeline requires input files in order to run. The only required input, aside from the sample(s), is a 
+reference genome.  The pipeline can be sped up by specifying URLs for the reference index files, which are generated 
+with `bwa index` and `samtools faidx`.
+
+## General Usage
  
-## Output
+1. Type `toil-bwa generate` to create an editable manifest and config in the current working directory.
+2. Parameterize the pipeline by editing the config.
+3. Fill in the manifest with information pertaining to your samples.
+4. Type `toil-bwa run [jobStore]` to execute the pipeline.
+
+## Example Commands
+
+Run sample(s) locally using the manifest
+1. `toil-bwa generate`
+2. Fill in config and manifest
+3. `toil-bwa run ./example-jobstore`
+
+Toil options can be appended to `toil-bwa run`, for example:
+`toil-bwa run ./example-jobstore --retryCount=1 --workDir=/data`
+
+For a complete list of Toil options, just type `toil-bwa run -h`
+
+Run a variety of samples locally
+1. `toil-bwa generate-config`
+2. Fill in config
+3. `toil-bwa run ./example-jobstore --retryCount=1 --workDir=/data --sample \
+    test-uuid file:///full/path/to/read1.fq.gz file:///full/path/to/read2.fq.gz`
 
-This pipeline produces a BAMFILE for a given sample.
+## Example Config
 
-## Running / Help
 
-It is recommended to use the associated launch scripts which provide default arguments needed to run the pipeline. 
-It is likely that the job store positional argument, `--workDir`, and `--output-dir` arguments will need to be modified.
-To run a pipeline after dependencies have been installed, simply:
+    # BWA Alignment Pipeline configuration file
+    # This configuration file is formatted in YAML. Simply write the value (at least one space) after the colon.
+    # Edit the values in this configuration file and then rerun the pipeline: "toil-bwa run"
+    # URLs can take the form: http://, file://, s3://, gnos://.
+    # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank
+    ##############################################################################################################
+    # Required: Reference fasta file
+    ref: s3://cgl-pipeline-inputs/alignment/hg19.fa
+    
+    # Required: Output location of sample. Can be full path to a directory or an s3:// URL
+    output-dir: /data/
+    
+    # Required: The library entry to go in the BAM read group.
+    library: Illumina
+    
+    # Required: Platform to put in the read group
+    platform: Illumina
+    
+    # Required: Program Unit for BAM header. Required for use with GATK.
+    program_unit: 12345
+    
+    # Required: Approximate input file size. Should be given as [integer][TGMK]
```

Comment:
The prefixes will _likely_ be obvious to most readers, but you might want to give an example. Also, clarify whether this is base-10 (Giga) or base-2 (Gibi), bits, bytes, etc.